src/OSPSuite.Presentation/Presenters/Importer/IModalImporterPresenter.cs

```diff
@@ -41,10 +41,13 @@ public ModalImporterPresenter(IModalImporterView view, IImporterPresenter import
          AddSubPresenters(importerPresenter);
       }
 
+      private bool _shouldPromtBeforeClose = false;
```

Comment:
move the definition of this flag at the top (use resharper) and correct typo (Prompt)